samples/multi-agent/src/main/java/demo/multiagent/application/EvaluatorAgent.java

```diff
@@ -82,7 +82,7 @@ public Effect<EvaluationResult> evaluate(EvaluationRequest request) {
     return effects()
       .systemMessage(SYSTEM_MESSAGE)
       .userMessage(evaluationPrompt)
-      .responseAs(EvaluationResult.class) // <3>
+      .responseWithJsonSchema(EvaluationResult.class) // <3>
```

Comment:
The prompt in this agent is still describing the schema.

Reply:
I kept that for additional clarity, showing how to give such instructions too. Also, schema doesn't work with all models so if someone take the example and switch model it will not work.